wolfhsm/wh_transport_mem.h

```diff
@@ -126,6 +127,7 @@ typedef struct {
     int initialized;
     uint16_t req_size;
     uint16_t resp_size;
+    uint16_t* cancel_seq;
```

Comment:
How come this is a pointer, vs just a member of the `whTransportMemContext` struct? AFAICT there isn't a reason it needs this indirection, and the cancellation state could be owned/held by the transport, but could be missing something